dumpling/export/writer_util.go

```diff
@@ -457,7 +456,7 @@ func buildFileWriter(tctx *tcontext.Context, s storage.ExternalStorage, fileName
 func buildInterceptFileWriter(pCtx *tcontext.Context, s storage.ExternalStorage, fileName string, compressType storage.CompressType) (storage.ExternalFileWriter, func(context.Context)) {
 	fileName += compressFileSuffix(compressType)
 	var writer storage.ExternalFileWriter
-	fullPath := path.Join(s.URI(), fileName)
+	fullPath := s.URI() + "/" + fileName
```

Comment:
is any wrong with `path.Join`？

Reply:
Customers are confused about this file path in log. So I choose to keep the same with BR external storage here:

https://github.com/pingcap/tidb/blob/7309b08/br/pkg/storage/s3.go#L501

```go
	fmt.Println(path.Join("s3://bucket", "test"))
	fmt.Println("s3://bucket" + "/" + "test")
```
```bash
s3:/bucket/test
s3://bucket/test
```